### DISCUSSION TITLE
Seq.App.EmailPlus + Source.ContributingEvents

### DISCUSSION THREAD

**Discussion by JHorvath-MaxetaTech** [datalust/seq-tickets#1755] [Help] [answered]
How can I display the message property of an event in my email body using Seq.AppEmailPlus + Source.ContributingEvents in an Alert?

I'm a little lost with Handlebars, even after reviewing the default template on GitHub (which I used verbatim, and can see is able to display the property).

I know this is the section that is outputs Source as one of the properties:

`
  {{#each $Properties}}
    <div style="margin-top:10px;" >
      <div title="{{@key}}" style="font-family:monospace;font-weight:bold;" >{{@key}}</div>
      <div style="font-family:monospace;word-wrap:break-word;white-space:pre;white-space:pre-wrap;" >{{pretty this}}</div>
    </div>
  {{/each}}
`

but is there some way to do **{{Source.ContributingEvents[0].Message}}** to just get the first event message?

The Alert documentation wasn't very clear on this...

**Comment by nblumhardt** [comment]
Hi John!

I think Handlebars uses plain `.` property access for array indexes, so it's something like:

`{{Source.ContributingEvents.0.2}}`

The `.0` is the first contributing event, and since the contributing events are just "columns" and the message is in the third column, it's `.2` to get the message property out.

Let me know if this does the job for you!

Nick

**Comment by JHorvath-MaxetaTech** [comment, marked as answer]
That got me on the right track, looks like it actually uses `.` and `[]` together:

`{{Source.ContributingEvents.[1].[2]}}`

and since `.[0]` are the column names, I needed to start at `.[1]` to get the data, but it works as expected.

**Reply by nblumhardt** [reply]
Great! Thanks for sharing the correct info 👍
